cloud/ec2_test.go

```diff
@@ -1160,14 +1162,9 @@ func (s *EC2Suite) TestGetRegion() {
 	s.Equal(evergreen.DefaultEC2Region, r)
 
 	(*s.h.Distro.ProviderSettings)["region"] = evergreen.DefaultEC2Region
-	s.NoError(ec2Settings.fromDistroSettings(s.h.Distro))
+	s.NoError(ec2Settings.FromDistroSettings(s.h.Distro, evergreen.DefaultEC2Region))
 	r = ec2Settings.getRegion()
 	s.Equal(evergreen.DefaultEC2Region, r)
-
-	(*s.h.Distro.ProviderSettings)["region"] = "us-west-2"
```

Comment:
I'm deleting this case for now because GetRegion will need to be changed a bit anyway, and for now this case should fail.